feat(2691): pass type refs to llm (#2712)
tailcallhq/tailcall: src/cli/llm/prompts/infer_type_name.md

> Given the sample schema of a GraphQL type, suggest 5 meaningful names for it.

Reviewer: ```suggestion
Given the sample schema of a GraphQL type, suggest {{count}} meaningful names for it.
```

We could move count into a template.

The name should be concise and preferably a single word.

> In this context, 'type_refs' refer to the number of times the type is used in different fields as output type. For example, if the following type is referenced 12 times in the 'user' field and 13 times in the 'profile' field.

Reviewer: ```suggestion
In this context, 'references' refer to the number of times the type is used in different fields as output type. For example, if the following type is referenced 12 times in the 'user' field and 13 times in the 'profile' field.
```

> While suggesting the type name, do consider above type_refs information for understanding the type context.

Reviewer: ```suggestion
While suggesting the type name, do consider above references information for understanding the type context.
```
